src/helpers/setup/CoGatewayHelper.js

```diff
@@ -197,23 +74,18 @@ class CoGatewayHelper {
     txOptions = defaultOptions;
```

Comment:
That is in the old (deprecated) helpers, and since they were in the middle of the parameter list I wasn't able to remove them without possibly breaking compatibility.

Also removing the code inside, which assigns to the function parameter wasn't possible, as that broke integration tests. It's really ugly, but in some places that the `*RawTx` variant sets the default options like gas on the `txOptions` that are passed in, and after that the `txOptions` are passed to the `sendTransaction`. I don't know how we could handle that in another way before removing it after deprecation, but I'm open to suggestions there.